nvflare/lighter/impl/static_file.py

```diff
@@ -20,10 +20,11 @@
 
 
 class StaticFileBuilder(Builder):
-    def __init__(self, enable_byoc=False, config_folder="", docker_image=""):
+    def __init__(self, enable_byoc=False, config_folder="", config_validator="", docker_image=""):
```

Comment:
Let's call it app_validator, instead of config_validator. App is the term we promote.

Reply:
Yes.  Just renamed this and other places one min ago.